.github/workflows/release.yaml

```diff
@@ -33,7 +33,7 @@ jobs:
       - name: Bump and Collect Version
         run: |
           pnpm changeset version
-          echo "BUILD_VERSION=$(sed -nE 's/^\s*"version": "(.*?)",$/\1/p' design-system/react/package.json)" >> $GITHUB_ENV
+          echo "BUILD_VERSION=$(sed -nE 's/^\s*"version": "(.*?)",$/\1/p' packages/app/package.json)" >> $GITHUB_ENV
```

Comment:
We should split this fixes in small PRs: https://github.com/FuelLabs/fuels-wallet/pull/255

Reply:
I just fix because I saw it